stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -637,8 +641,10 @@ public class MessageListView : ConstraintLayout {
         messageListItemViewHolderFactory.decoratorProvider = MessageListItemDecoratorProvider(
             currentUser = currentUser,
             dateFormatter = messageDateFormatter,
-            isDirectMessage = channel.isDirectMessaging()
+            isDirectMessage = channel.isDirectMessaging(),
+            messageListViewStyle.itemStyle
```

Comment:
```suggestion
            messageListViewStyle.itemStyle,
```

Reply:
Should these be added automatically when I cmd+option+l? It does add some in the signatures, but not the calls 🤔

Reply:
The formatter isn't quite as aggressive with them as it should be 😄